project/models.py

```diff
@@ -351,7 +351,9 @@ def can_raise_pr(self, contributor):
     # TODO: ISSUE: Rename this function to 'get_deadline' as it is more suitable. Don't Forget to update name at all
     #  places.
     def get_remaining_time(self):
-        return self.assigned_at + timezone.timedelta(days=self.issue.get_issue_days_limit())
+        datetime = self.assigned_at + timezone.timedelta(days=self.issue.get_issue_days_limit())
+        local_dt = timezone.localtime(datetime, timezone.get_fixed_timezone(330))
```

Comment:
Add comments here, what exactly it is returning

Reply:
done